GettingStarted.md

```diff
@@ -0,0 +1,181 @@
+The instructions below will help you get your development environment configured so that you can build, test and debug Team Explorer Everywhere.
+
+## Clone and configure the repository
+Use the Git tool of your choice to clone the repository into a local path.
+For example, you could use git.exe from a Windows console window:
+```
+mkdir c:\repos
+pushd c:\repos
+git clone https://github.com/Microsoft/team-explorer-everywhere.git
+```
+
+We keep LF line-ending on the server. Please set the `core.safecrlf` git config property to `true`.
+```
+git config --local core.safecrlf true
+```
+
+## Install development tools
+
+### Install Java
+
+Two versions of the Java Development Kit are needed: we use Java SE 6 as the minimal supported Java execution environment and to build the release kit, as well as Java SE 8 for running Eclipse.
+
+1. Download and install Java SE 6:
+  * [Windows and Linux: Java Development Kit 6u45](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase6-419409.html#jdk-6u45-oth-JPR)
+  * [Mac: Apple Java 1.6.0_65, a.k.a. Java for OS X 2015-001](https://support.apple.com/kb/DL1572?locale=en_US)
+1. Set the JAVA_HOME environment variable to point to the install, e.g
+ * (Windows) `SET JAVA_HOME=C:\Program Files\Java\jdk1.6.0_45`
+ * (Linux) `JAVA_HOME=~/dev/java/jdk1.6.0_45`
+ * (Mac) `JAVA_HOME=/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home`
+1. Add JAVA_HOME bin directory to the path
+ * (Windows) `SET PATH=%JAVA_HOME%\bin;%PATH%`
+ * (Linux) `PATH=$JAVA_HOME/bin:$PATH`
+ * (Mac) `PATH=$JAVA_HOME/bin:$PATH`
+1. Download and install Java SE 8 JDK from [Java SE Downloads](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+### Install Ant
+
+1. If you do not already have it, download and install Apache Ant(TM) version 1.9.6 from [Ant Binary Distributions](http://ant.apache.org/bindownload.cgi).
+1. Add the full path of the Ant `bin` directory to the `PATH` system environment variable. You can find more Ant installation details [here](http://ant.apache.org/manual/install.html#installing).
+ * (Windows) `SET PATH=C:\dev\apache-ant-1.9.6\bin;%PATH%`
+ * (Linux) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
+ * (Mac) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
+
+### Install Eclipse
+
+Two versions of Eclipse are needed: we use Eclipse 3.5.2 (Galileo) as the Target Environment, which is the minimum supported Eclipse version (used for building the release and testing backward-compatibility) and Eclipse 4.5.2 (Mars.2) as a dedicated team-explorer-everywhere Development Environment.  Using a dedicated Eclipse installation and workspace for development makes it less disruptive to configure Eclipse-wide preferences.
+
+1. Download and extract [Eclipse Classic 3.5.2](http://www.eclipse.org/downloads/packages/eclipse-classic-352/galileosr2) to the `dev/eclipseTargets/352` sub-folder under your `HOME` folder.  The absolute path to this folder will be needed later when building with Ant.
```

Comment:
1. Why do we first suggest to use the long root path for the Eclipse installation putting the user into troubles with long paths on Windows, and then provide him with the workaround for the artificially created issue? ;-) Looks rather strange for me.
2. Reading the this instruction I feel that I **HAVE** to use `dev/eclipseTarget/352` because it's hardcoded somewhere in TEE code/scripts. May we provide somewhat softer sentence? Say "... _for example_ C:\Eclipse352 or ~/dev/eclipse/352..."?